test/distributed/org/apache/cassandra/distributed/test/guardrails/GuardrailCollectionSizeOnSSTableWriteTest.java

```diff
@@ -434,4 +436,16 @@ private String failMessage(String key)
     {
         return String.format("Detected collection v in row %s in table %s of size", key, qualifiedTableName);
     }
+
+    private void createTable(String cql)
```

Comment:
why replace schemaChange w/ createTable which disables compaction? The tests seem to pass ok without this change

Reply:
also applies to `SSTableIdGenerationTest.java` and `GuardrailItemsPerCollectionOnSSTableWriteTest.java`

Reply:
If I recall these tests didn't actually disable auto-compaction like they thought they were because disabling auto-compaction on a table that hasn't been created is a no op! The auto-compaction config is on the table not the keyspace.

I don't recall what got me looking at these tests in the first place. They might have just failed randomly on me.

It's renamed `createTable` because that is what it was used for, although technically it runs any `cql` and then does the auto-compaction change. Definitely open to changing the name to something like `